llvm/test/CodeGen/RISCV/rvv/fixed-vectors-interleaved-access.ll

```diff
@@ -1782,7 +1783,7 @@ define void @store_factor4_one_active_fullwidth(ptr %ptr, <16 x i32> %v) {
 ; CHECK-LABEL: store_factor4_one_active_fullwidth:
 ; CHECK:       # %bb.0:
 ; CHECK-NEXT:    li a1, 16
-; CHECK-NEXT:    vsetivli zero, 4, e32, m4, ta, ma
+; CHECK-NEXT:    vsetivli zero, 4, e32, m1, ta, ma
```

Comment:
I guess this has something to do with the extra ExtractVector you created on the first operand of shufflevector.

Reply:
Yep, this is the optimization mentioned in the review description.